staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -106,7 +152,7 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 			cost := uint64(math.Ceil(float64(actualSize(args[0])) * 2 * common.StringTraversalCostFactor))
 			return &cost
 		}
-	case "masked", "prefixLength", "family", "isUnspecified", "isLoopback", "isLinkLocalMulticast", "isLinkLocalUnicast":
+	case "masked", "prefixLength", "family", "isUnspecified", "isLoopback", "isLinkLocalMulticast", "isLinkLocalUnicast", "isGlobalUnicast":
```

Comment:
`isGlobalUnicast` was just missing before?

Reply:
yeah, got caught by the test-only fail-on-missing-cost-estimate change I made

we got lucky that the fallback "this costs 1" was actually reasonable for this setter